### DISCUSSION TITLE
Daily 17/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#305] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Realicé algunas refactorizaciones de código en Angular y Django. Hoy tengo Ingles, tengo Java, y tengo que terminar una clase pendiente de Python.

**Comment by NataliaAlvarezIspc** [comment]
Ayer continué con la tarea de crear perfil, le agregué un  formulario para cambiar algunos datos  pero no puedo lograr que me tome las validaciones. También estuve leyendo algunas páginas para agregar la Api de prueba.
Hoy estaré instalando django y revisando la pagina de Perfil. Además trataré de terminar con la lectura de los apuntes de la plataforma. Por la noche tengo clase de Inglés del Instituto.
